// api/mockBook.js
export default [
    {
        id: "1",
        categoryId: "1",
        name: "THE BEATLES",
        description: "LET IT BE / 1970",
        price: 90,
        image: '/uploads/pasted.png',

    },

    {
        id: "2",
        categoryId: "2",
        name: "THE ROLLING STONES",
        description: "AFTER MATH / 1966",
        price: 75,
        image: '/uploads/rolling.png',


    },

    {
        id: "3",
        categoryId: "3",
        name: "ALICE COOPER",
        description: "WELCOME TO MY NIGHTMARE",
        price: 60,
        image: '/uploads/alice.png',

    },

    {
        id: "4",
        categoryId: "4",
        name: "AEROSMITH",
        description: "AEROSMITH / 1970",
        price: 80,
        image: '/uploads/aero.png',
        pages: 5
    },

    {
        id: "5",
        categoryId: "5",
        name: "WHITESNAKE",
        description: "THE PURPLE ALBUM / 2016",
        price: 50,
        image: '/uploads/snake.png',

    },

    {
        id: "6",
        categoryId: "6",
        name: "DEEP PURPLE",
        description: "MACHINE HEAD /1970 ",
        price: 45,
        image: '/uploads/deep.png',

    },

    {
        id: "7",
        categoryId: "1",
        name: "THE BEATLES",
        description: "THE REVOLVER/ 1965",
        price: 70,
        image: '/uploads/item.png',

    },

    {
        id: "8",
        categoryId: "1",
        name: "THE BEATLES",
        description: "ABBEY ROAD/ 1966",
        price: 70,
        image: '/uploads/abby.png',

    },
    {
        id: "9",
        categoryId: "1",
        name: "THE BEATLES",
        description: "WITH THE BEATLES / 1966",
        price: 70,
        image: '/uploads/with.png',

    },
    {
        id: "10",
        categoryId: "1",
        name: "THE BEATLES",
        description: "HELP! / 1963",
        price: 70,
        image: '/uploads/help!.png',

    },
    {
        id: "11",
        categoryId: "1",
        name: "THE BEATLES",
        description: "A HARD DAYS NIGHT / 1963",
        price: 70,
        image: '/uploads/hard.png',

    },
    {
        id: "12",
        categoryId: "2",
        name: "THE ROLLING STONES",
        description: "SOME GIRLS / 1966",
        price: 75,
        image: '/uploads/roll.png',

    }

]
